fades/main.py

```diff
@@ -253,6 +255,12 @@ def go(argv):
         create_venv = True
 
     if create_venv:
+        # Check if the requested packages exists in pypi.
+        if not args.no_precheck_availability:
+            if not helpers.check_pypi_exists(indicated_deps):
+                logger.info("A indicated dependency doesn't exists. Exiting")
```

Comment:
Some details here:

- s/A indicated/An indicated/

- this situation is serious enough to make fades quit... maybe we should log in WARNING?

- wouldn't it be great if we could log here (or in the called function) *which dependency* is the one generating this situation?

Reply:
- typo fixed 
- I think if the user is asking for a dependency that doesn't exists we should fail ASAP. (it's maybe a typo) 
- we are currently showing it 

```
 ~/code/fades   precheck-availability ●  bin/fades -d sarasa                                                                                                 123476ms 
*** fades ***  2018-03-17 18:17:30,636  INFO     sarasa doesn't exists in PyPI.
*** fades ***  2018-03-17 18:17:30,637  INFO     An indicated dependency doesn't exists. Exiting
```

Reply:
FTR: I will change the loglevel to ERROR